src/include/duckdb/execution/operator/persistent/physical_copy_to_file.hpp

```diff
@@ -37,6 +37,7 @@ class PhysicalCopyToFile : public PhysicalOperator {
 	bool per_thread_output;
 	optional_idx file_size_bytes;
 	bool rotate;
+	bool return_files;
```

Comment:
Maybe we can make this an enum instead of a boolean (e.g. `CopyReturnType::CHANGED_ROWS`, `CopyReturnType::FILE_LIST`)?

Reply:
I've made it so we get back the number of changed rows, AND the file list. I'll make an enum that clearly describes this.